dz13/src/app.js

```diff
@@ -0,0 +1,121 @@
+function isSomeProductInStock(productsList) {
+  return productsList.some((product) => product.inStock)
+}
+
+function makeUserTotalBalance(transactions) {
+  const newTransactions = transactions.reduce((result, transaction) => {
+    if (transaction.type === 'income') {
+      return result + transaction.amount
+    }
+    return result - transaction.amount
+  }, 0)
+  return newTransactions
+}
+
+function entriesArray(array) {
```

Comment:
```suggestion
function flattenArray(array) {
```

начинай название функции с глагола